### ISSUE TITLE
Maintainership status of the project

### ISSUE THREAD

**Issue by vi** [DanielKeep/cargo-script#71]
Is it maintained by original developer?

If no, it may be worth to explicitly mark it as unmaintained in README; or invite some other maintainer (i.e. convert Github repo to org and add contributors, add owners on crates.io).

**Comment by reitermarkus** [comment]
Given that @DanielKeep hasn't been active on GitHub in over a year, I don't think we will get a response here anytime soon.

I've started a fork at https://github.com/reitermarkus/cargo-eval. Please let me know if you want to be added as a maintainer, so it doesn't suffer the same fate.

**Comment by igor-petruk** [comment]
For people who are also concenred with maintenance status of this projects I suggest to take a look at https://github.com/igor-petruk/scriptisto, which works for any language, beyond Rust. It can also build in Docker without host compilers at all, extracting statically-linked C, C++, Rust, OCaml, and other binaries to the host system.

**Comment by sstadick** [comment]
@reitermarkus, Are github issues not on for your fork? Also, if you are interested in help maintaining, I would be more than happy to help.

**Comment by reitermarkus** [comment]
@sstadick, yeah, seems like issues are disabled by default. Enabled them now. Also seems like I completely missed the open PRs.

**Comment by bevsxyz** [comment]
[rust-script](https://github.com/fornwall/rust-script) seems to be the currently maintained fork.
